src/containers/pointsContainer.js

```diff
@@ -211,7 +211,7 @@ const PointsContainer = ({
   const _getESTMPrice = (points) => {
     const { base, quote } = globalProps;
 
-    return points * 0.01 * (base / quote);
+    return points * 0.00442 * (base / quote);
```

Comment:
What we did in website, `points/150` that would give estimation non-related to HIVE price (base/quote) only for Boosts though. Not for Wallet points estimated value.